### ISSUE TITLE
dataloader.py - what'a the output of this "labels"?

### ISSUE THREAD

**Issue by michaeldengxyz** [minus31/BlazeFace#2]
Inside dataloader.py:
------------------
    img_files = glob.glob(dataset_dir)
    img_files = [f for f in img_files if f[-3:] in IM_EXTENSIONS]

    with open(label_path, "rb") as f:
        labels = pickle.load(f)
    
    numofData = len(img_files)# endwiths(png,jpg ...)
    data_idx = np.arange(numofData)
-----------------
 labels = pickle.load(f)   
what'a the output of this array "labels"?

**Comment by minus31** [comment]
The labels are differed by datasets and how you define the anchor boxes. 
I didn't make this repo to fit in a specific dataset. If you propose a dataset for an example, I would be able to answer your question.

**Comment by HangJie720** [comment]
Hello, I’d like to ask you something about training data and labels. What data do you use to train the model? Is the link available ?

**Comment by minus31** [comment]
@HangJie720 

Hello, I used datasets that I make on my own. This code is kind of template to utilize for various datasets. Thank you for asking!

**Comment by Jaren1907** [comment]
> The labels are differed by datasets and how you define the anchor boxes.
> I didn't make this repo to fit in a specific dataset. If you propose a dataset for an example, I would be able to answer your question.

Hello, I have a dataset made by myself. the label is like (x, y, w, h), which means the left-top point, width and height of face in picture. I wonder how should I set the format of 'labels' in
```
with open(label_path, "rb") as f:
    labels = pickle.load(f)
```
I would appreciate it if you give me a little example, Thank you !

**Comment by minus31** [comment]
@Jaren1907
 Basically, BlazeFace uses Anchor box. Hence, the label form should follow Anchor scheme.
To make anchor boxes, I recommend you to refer RetinaNet repo (https://github.com/fizyr/keras-retinanet)

**Comment by suyali** [comment]
@minus31   I stil have no idea about the setting of labels,  can you give me an example?

**Comment by wangm-word** [comment]
Hello, I have a dataset made by myself. the label is like (x, y, w, h), which means the left-top point, width and height of face in picture. I wonder how should I set the format of 'labels' in，Can you give details？thank you！
